### ISSUE TITLE
Ament_python not found

### ISSUE THREAD

**Issue by LoyVanBeek** [mikaelarguedas/ros2ci#24]
I've been learning some ROS2 lately and (as the good boy my parent raised me) wanted to setup CI for it so started using this repo. 

However, I get this error:

```
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
telegram_ros2: Cannot locate rosdep definition for [ament_python]
```
(See https://travis-ci.org/github/LoyVanBeek/telegram_ros2/jobs/663494148#L1100)

Now I'm confused: `ament_python` is the `buildtool_depend` since this is a Python pkg and started with `ros2 pkg create --build-type ament_python <package_name>` in accordance with https://index.ros.org/doc/ros2/Tutorials/Creating-Your-First-ROS2-Package/. 

But `ament_python` never mentioned in https://github.com/ros/rosdistro/blob/93d6478cf9ec083780c03cc58821915a3b057cf9/eloquent/distribution.yaml#L9 and the [repo](https://github.com/ament/ament_python) is empty too. 

So what is off here? I'm not even sure this is the right place to ask, sorry if not.

**Comment by mikaelarguedas** [comment]
> Now I'm confused: ament_python is the buildtool_depend since this is a Python pkg and started with ros2 pkg create --build-type ament_python <package_name> in accordance with https://index.ros.org/doc/ros2/Tutorials/Creating-Your-First-ROS2-Package/.

TL;DR `ament_python` is a build type and not a ROS package, so it should *not* be listed as a dependency. Removing the line `<buildtool_depend>ament_python</buildtool_depend>` should fix the issue.

---

This is a problem in `ros2 pkg create` that has been fixed but not yet propagated to all ROS distributions: https://github.com/ros2/ros2cli/pull/439

> I've been learning some ROS2 lately and (as the good boy my parent raised me) wanted to setup CI for it so started using this repo.

:+1: that is a very good initiative!

>  I'm not even sure this is the right place to ask, sorry if not.

I'd say ROS Answers is the best place to start if you're not sure here the issue is from. In most cases you'll get a correct answer, if it is an actual bug you'd get redirected to the repository where to open an issue.

**Comment by LoyVanBeek** [comment]
Thnx. I expected `colcon` to bork when removing that dependency but to my surprise it didn't and expected the generator to know better than me. 

Thanks for the quick reply!

**Comment by mikaelarguedas** [comment]
You're welcome, I'll close this under the assumption that removing the dependency solved your issue.

Feel free to comment here if it's not the case and we can reopen
